cli/src/commands/sync.ts

```diff
@@ -20,6 +21,23 @@ export default {
     }
   ) =>
     scan(opts).pipe(
+      switchMap(async (account) =>
+        account.nfts?.length
+          ? {
+              ...account,
+              nfts: await Promise.all(
+                account.nfts.map(async (nft) => {
+                  const { result: metadata } = await metadataCallBatcher.load({
+                    contract: nft.collection.contract,
+                    tokenId: nft.tokenId,
+                  });
+
+                  return { ...nft, metadata };
+                })
```

Comment:
mmh i have a feeling that logic should be generalized in a function back into live-common to "resolve all lazy fields of an account" because that logic feels specific